### ISSUE TITLE
Accessibility: contrast ratio - gov-phase-banner/alpha

### ISSUE THREAD

**Issue by K-T-O** [CJSCommonPlatform/govuk_single_page_pdk#104]
Pa11y files an error about the following issue:

- WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail

Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.46:1.

This occurs in a gov-phase-banner/alpha component.

**Comment by fabiotisci** [comment]
This styles come from the gov elements from GDS, UX are aware
